TablesLists/iOSTableAndCellStyles/Info.plist

```diff
@@ -26,5 +26,7 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>CFBundleIdentifier</key>
+	<string>com.xamarin.tables</string>
```

Comment:
there are many table-related samples for iOS, i think a more unique identifier for each one is necessary as customers are likely to install them all as they work through the documentation